app/javascript/gobierto_data/lib/factories/visualizations.js

```diff
@@ -29,5 +29,8 @@ export const VisualizationFactoryMixin = {
     deleteVisualization(id) {
       return axios.delete(`${endPoint}/${id}`, { headers });
     },
+    getListVisualizations() {
```

Comment:
What's the difference between this and `getVisualizations` with no params?

Reply:
`getVisualizations` with no params gets the entire list of visualizations, for example: [Visualizations](https://getafe.gobify.net/api/v1/data/visualizations)

[Gobierto Data API](https://gobierto.readme.io/docs/api-gobierto-data#list-visualizations)

Reply:
What I mean is the actual axios call is the same in both functions
```
    getVisualizations(params) {
      const qs = new URLSearchParams(params)
      return axios.get(`${endPoint}?${qs.toString()}`, { headers });
    },
    getListVisualizations() {
      return axios.get(`${endPoint}`, { headers });
    },
```
If `getVisualizations` has no params, it returns the same as `getListVisualizations`. Therefore, `getListVisualizations` is pointless, use the previous one with no params instead.